rest-service/manager_rest/test/resources/deployment_update/depup_step/setup_with_plugin_1.yaml

```diff
@@ -0,0 +1,33 @@
+tosca_definitions_version: 'cloudify_dsl_1_3'
+
+plugins:
+  should_uninstall:
+    executor: central_deployment_agent
+    source: .
+
+  should_skip_install:
+    executor: central_deployment_agent
+    source: .
+    package_version: "1"
+
+  should_reinstall:
```

Comment:
What will happen if a tried to put here a plugin with version 2 and update with version 1? this needs also a test

Reply:
Added, FYI the changes are detected by comparing the two dicts.